packages/front-end/types/app-features.ts

```diff
@@ -0,0 +1,18 @@
+export type AppFeatures = {
+  "papercups-config": Record<string, unknown>;
+  "show-experiment-groups": boolean;
+  "show-experiment-targeting": boolean;
+  "experiment-implementation": boolean;
+  "feature-usage-code": boolean;
+  "feature-list-realtime-graphs": boolean;
+  chatwoot: Record<string, unknown>;
+  "test-experiment": number;
+  "webflow-ctas": boolean;
+  "guided-onboarding-test-august-2022": boolean;
+  "self-serve-billing": boolean;
+  "self-serve-billing-overage-warning-banner": boolean;
+  "mk-test-feature": boolean;
+  "secret-api-keys": boolean;
+  "stats-engine-setting": boolean;
+  "proxy-cloud": boolean;
+};
```

Comment:
These don't all look relevant to production GrowthBook but perhaps we can add project filtering to the REST API (if it's not there already) and the CLI.